### PR TITLE
Create a OmegaRef Parameters XML

### PR DESCRIPTION
**Pull request by marcosmc**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*
Feature

**What is the new behavior (if this is a feature change)?**
Create a parameter XML file with the OmegaRef parameters, all the generators connected to the OmegaRef have the same weight until the end of the development that allows reading the parameters of the dynamic models.

### CODE REVIEW

File: src/main/java/com/powsybl/dynawo/xml/ParXml.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                }
+            }
+            if (index >= 0) {
+                writeParameter(writer, "INT", "nbGen", Integer.toString(index + 1));
+                writer.writeEndElement();
+            }
+        } catch (XMLStreamException e) {
+            throw new UncheckedXmlStreamException(e);
+        }
+    }
```
Comment (agnesLeroy) [inline src/main/java/com/powsybl/dynawo/xml/ParXml.java:69]: Here it seems to me you're not following exactly the Dynawo serialization: generators are numbered from 0 to N-1 in the example DYD file (https://github.com/dynawo/powsybl-dynawo-data/blob/master/IEEE14_SyntaxExamples/IEEE14.dyd). Why are you numbering using `index+1`?

Comment (marcosmc) [reply src/main/java/com/powsybl/dynawo/xml/ParXml.java:69]: The parameter with the weight of the generators is numbered from 0 to N - 1 (index = xmlContext.getIndex(dynawoModel.getLib(), true) starts at 0), and for the parameter nbGen, where the number of generators connected to the OmegaRef (N) is specified, the last index + 1 is used.

File: src/main/java/com/powsybl/dynawo/xml/ParXml.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+                    // FIXME Change it by gen.H * gen.SNOM
+                    // when parameters from generator dynamic model are available
+                    double weight = OMEGA_REF_TEMP_WEIGHT_NOT_CALCULATED;
+                    writeParameter(writer, "DOUBLE", "weight_gen_" + index, Double.toString(weight));
+                }
+            }
+            if (index >= 0) {
+                writeParameter(writer, "INT", "nbGen", Integer.toString(index + 1));
+                writer.writeEndElement();
+            }
```
Comment (mathbagu) [inline src/main/java/com/powsybl/dynawo/xml/ParXml.java:65]: I'm not sure this is the best way to implement this. I think each model should be responsible to serialize its parameters, with the same design than the DYD

Comment (marcosmc) [reply src/main/java/com/powsybl/dynawo/xml/ParXml.java]: Moved

**Pull request merged**